Formula/varnish.rb

```diff
@@ -28,10 +28,17 @@ class Varnish < Formula
   def install
     ENV["PYTHON"] = Formula["python@3.9"].opt_bin/"python3"
 
+    # flags to set the paths used by varnishd to load VMODs and VCL,
+    # pointing to the ${HOMEBREW_PREFIX}/ shared structure so other packages
+    # can install VMODs and VCL.
+    make_cflags = %W[
+      -DVARNISH_VMOD_DIR='"#{HOMEBREW_PREFIX}/lib/varnish/vmods"'
+      -DVARNISH_VCL_DIR='"#{pkgetc}:#{HOMEBREW_PREFIX}/share/varnish/vcl"'
+    ]
```

Comment:
```suggestion
    ENV.append_to_cflags "-DVARNISH_VMOD_DIR=#{HOMEBREW_PREFIX}/lib/varnish/vmods"
    ENV.append_to_cflags "-DVARNISH_VCL_DIR=#{pkgetc}:#{HOMEBREW_PREFIX}/share/varnish/vcl"
```

Reply:
No, these flags are used for *just the `make` command*. The flags should definitely not be added to the `ENV` environment, as that breaks all sorts of other things. Otherwise, I would have passed these to configure instead.

Reply:
Put differently: if I apply your requested changes, the build breaks, unfortunately, as the environment variables end up in `#define`s inside a generated include file setting the C compiler flags for the VCL DSL compiler, which should not have these flags.

(Also, the two variables are used as literal C strings, so the quoting on them is important. The variables need to retain their `"` quotes).

Reply:
Even if you do this after calling `configure`?

Reply:
🤦 Of course, I needed to do this after running `./configure`. That works just fine, and no adverse effects on the `.h` files either. Sorry about that!